### ISSUE TITLE
Visual issue on tours.html

### ISSUE THREAD

**Issue by mylinhphxn** [mylinhphxn/github-issue-template#9]
**Describe the bug**
Using the accessibility checker, I found there is a visual issue on the tours.html page. Frame present on the web page does not have title attribute.

**To Reproduce**
1. Go to https://www.accessibilitychecker.org/
2. Paste in https://mylinhphxn.github.io/github-issue-template/tours.html
3. Scroll down to "Required Manual Audits (NaaN)
4. It is the second visual issue listed 

**Expected behavior**
We should get an indication of no errors from the validator.

**Screenshots**
The errors if not fix can be viewed at the following URL: https://www.accessibilitychecker.org/audit/?website=https%3A%2F%2Fmylinhphxn.github.io%2Fgithub-issue-template%2Ftours.html&flag=us

**Desktop (please complete the following information):**
-N/A

**Smartphone (please complete the following information):**
-N/A

**Additional context**
Be sure to use the validator to identify further issues as applicable.
